src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.ts

```diff
@@ -57,13 +57,17 @@ export class ExerciseDetailsStudentActionsComponent implements OnInit, OnChanges
     routerLink: string;
     repositoryLink: string;
 
+    theiaEnabled: boolean = false;
+    theiaPortalURL: string = '';
```

Comment:
why dont you just write theiaPortalURL: string;

It would be consistent like the other variables above.

Reply:
Done